docs/APIs.md

```diff
@@ -8522,8 +8502,9 @@ have buffers to read.
 <em>(Optional)</em>
 <p>
 
-ReplicasPerScale defines maximum replicas can be scaled up or down at
-once. The is use to prevent too aggressive scaling operations
+DeprecatedReplicasPerScale defines maximum replicas can be scaled up or
+down at once. The is use to prevent too aggressive scaling operations
```

Comment:
```
DeprecatedReplicasPerScale defines the maximum number of replicas that can be scaled up or
down in a single scaling operation. The is used to prevent over aggressive scaling operations
```